ruff.toml

```diff
@@ -0,0 +1,91 @@
+target-version="py310"
```

Comment:
Is it ok to have it a top level? Shouldn't be in /python?

Reply:
I think we're going to follow the JS pattern and leave all the tool configs at the top, this way the language folders can remain plain package listings.